.github/CODEOWNERS

```diff
@@ -75,28 +75,28 @@ rush.json       @iTwin/itwinjs-core-admins
 /common/**/linear-referencing-common/         @calebmshafer @diegoalexdiaz
 /common/**/core-orbitgt.*                     @iTwin/itwinjs-core-display
 /common/**/core-orbitgt/                      @iTwin/itwinjs-core-display
-/common/**/perf-tools.*                       @calebmshafer @wgoehrig
-/common/**/perf-tools/                        @calebmshafer @wgoehrig
+/common/**/perf-tools.*                       @aruniverse @wgoehrig
+/common/**/perf-tools/                        @aruniverse @wgoehrig
 /common/**/physical-material-backend.*        @bbastings @jchick-bentley
 /common/**/physical-material-backend/         @bbastings @jchick-bentley
 /common/**/presentation-*.*                   @iTwin/itwinjs-core-presentation
 /common/**/presentation-*/                    @iTwin/itwinjs-core-presentation
-/common/**/rpcinterface-full-stack-tests/     @calebmshafer @wgoehrig
+/common/**/rpcinterface-full-stack-tests/     @aruniverse @paulius-valiunas @wgoehrig
 /common/**/appui-*.*                          @iTwin/itwinjs-core-ui
 /common/**/appui-*/                           @iTwin/itwinjs-core-ui
 /common/**/webgl-compatibility.*              @iTwin/itwinjs-core-display @Ellord207
 /common/**/webgl-compatibility/               @iTwin/itwinjs-core-display @Ellord207
 /common/**/core-webpack-tools/                @aruniverse @calebmshafer @wgoehrig
 
-/common/config/azure-pipelines  @calebmshafer @wgoehrig @ColinKerr @aruniverse
+/common/config/azure-pipelines  @aruniverse @calebmshafer @ColinKerr @wgoehrig
 
 /core/backend                   @iTwin/itwinjs-core-admins
 /core/bentley                   @iTwin/itwinjs-core-admins
 /core/common                    @iTwin/itwinjs-core-admins @iTwin/itwinjs-core-display
 /core/common/src/rpc/IModelTileRpcInterface.ts    @pmconne
 /core/ecschema-*                @calebmshafer @ColinKerr @wgoehrig
 /core/electron                  @wgoehrig @GytisCepk @ben-polinsky
-/core/express-server            @calebmshafer @wgoehrig
+/core/express-server            @paulius-valiunas @wgoehrig
 /core/extension                 @calebmshafer @wgoehrig @aruniverse @kckst8 @johnnyd710
```

Comment:
maybe remove @kckst8 @johnnyd710 from extensions? dont want to spam them, unless they want it :)